GGSmtp/GGSmtp.Tests/DataResponderTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NUnit.Framework;
+using Simple.MailServer.Smtp.Config;
+using Moq;
+using Simple.MailServer.Smtp;
+using System.IO;
+using Simple.MailServer;
+using System.Text;
+
+namespace GGSmtp.Tests
+{
+    [TestFixture]
+    public class DataResponderTests
```

Comment:
Our conventions might not be that clear, but a unit test should be named after the class under test with a Test(s) suffix.
So it should be named DefaultDataResponderTests !fix